.travis.yml

```diff
@@ -23,6 +23,8 @@ install:
   # Warm up some caches so that our tests don't timeout
   - sandbox run 2.0.0 -f visualization -f monitoring > /dev/null
   - conduct load -q cassandra
+  - conduct load -q cinnamon-grafana-docker:v2
```

Comment:
A few more bundles had to be preloaded - we didn't notice before because errors were ignored.